src/MagnumPlugins/AssimpImporter/AssimpImporter.cpp

```diff
@@ -804,25 +815,63 @@ Containers::Pointer<AbstractMaterialData> AssimpImporter::doMaterial(const Unsig
     /* Key always present, default black */
     aiColor4D diffuseColor;
     UnsignedInt diffuseTexture{};
+    UnsignedInt diffuseCoordinateSet{};
     mat->Get(AI_MATKEY_COLOR_DIFFUSE, diffuseColor);
-    if(flags & PhongMaterialData::Flag::DiffuseTexture)
+    Debug() << "Hello Material";
+    if(flags & PhongMaterialData::Flag::DiffuseTexture) {
         diffuseTexture = firstTextureIndex++;
+        mat->Get(AI_MATKEY_UVWSRC(aiTextureType_DIFFUSE, 0), diffuseCoordinateSet);
+
+        if(diffuseCoordinateSet != 0) {
```

Comment:
```suggestion
        /* Couldn't convince Assimp to import non-zero values from a COLLADA 
           file, so this code path isn't tested */
        if(diffuseCoordinateSet != 0) {
```

Add an explanatory comment so we know *why* this isn't covered. Same in the two cases below.